releases/evm/2025-02-xrplevm-GMP-v6.4.0.md

```diff
@@ -0,0 +1,237 @@
+# Ripple EVM Sidechain GMP v6.4.0
+
+|                | **Owner**                                                                   |
+| -------------- | --------------------------------------------------------------------------- |
+| **Created By** | @blockchainguyy <ayush@interoplabs.io>                                      |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io>, @talalashraf <talal@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
+| **Testnet**          | -                     | TBD      |
+| **Mainnet**          | -                     | TBD      |
+
+- [Releases](https://github.com/axelarnetwork/axelar-cgp-solidity/tree/v6.4.0)
+
+## Background
+
+Changes in the release:
+
+This is the v6.4.0 deployment of EVM compatible Amplifier Gateway contracts for Ripple Sidechain.
+
+## Deployment
+
+Create a `.env` config. `CHAIN` should be set to `xrplevm` for all production networks: `mainnet`, `testnet`, `stagenet` and `devnet-amplifier`.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAIN=xyz
+```
+
+An initial `xrplevm` chain config needs to be added to `${ENV}.json` file under `chains` key.
+
+#### Devnet-Amplifier / Stagenet / Testnet
+
+```json
+"xrplevm": {
+    "name": "Ripple EVM",
+    "axelarId": "xrplevm",
+    "chainId": 1440002,
```

Comment:
@blockchainguyy let's confirm the right chainId for testnet (`1440002` is the devnet)

Reply:
run the RPC compat tests to ensure params are set fine